api/envoy/api/v2/cds.proto

```diff
@@ -385,6 +385,11 @@ message Cluster {
     // going to an individual locality if said locality is disproportionally affected by the
     // subset predicate.
     bool scale_locality_weight = 5;
+
+    // If true, when a fallback policy is configured and it fails to find a host this will
```

Comment:
I'd reword this to make it clear that it's the subset LB that is failing to find a host according to the fallback policy, right now it sounds like the policy itself is failing which seems a bit misleading.

Reply:
Done, lmk if it sounds better now.